apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.ejs

```diff
@@ -17,58 +17,147 @@
-                                    <% if (authz_data.has_course_permission_edit && (! course.example_course)) { %>
-                                        <span class="text-danger">You must <a href="<%= urlPrefix %>/<%= navPage %>/syncs">sync your course</a> before viewing or editing its configuration</span>
-                                    <% } else { %>
-                                        <span class="text-danger">A course editor must sync this course before anyone can view or edit its configuration</span>
-                                    <% } %>
-                                <% } else if (locals.noInfo) { %>
-                                    <span class="text-danger">Missing configuration file</span>
-                                    <% if (authz_data.has_course_permission_edit && (! course.example_course)) { %>
-                                        <form name="add-configuration-form" class="d-inline" method="POST">
-                                            <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
-                                            <button name="__action" value="add_configuration" class="btn btn-xs btn-secondary mx-2">
-                                                <i class="fa fa-edit"></i>
-                                                <span class="d-none d-sm-inline">Create infoCourse.json</span>
-                                            </button>
-                                        </form>
-                                    <% } %>
-                                <% } else { %>
-                                    <% if (authz_data.has_course_permission_view) { %>
-                                        <a href="<%= urlPrefix %>/<%= navPage %>/file_view/infoCourse.json">
-                                            infoCourse.json
-                                        </a>
-                                        <% if (authz_data.has_course_permission_edit && (! course.example_course)) { %>
-                                            <a tabindex="0" class="btn btn-xs btn-secondary mx-2" role="button" href="<%= urlPrefix %>/<%= navPage %>/file_edit/infoCourse.json">
-                                                <i class="fa fa-edit"></i>
-                                                <span>Edit</span>
-                                            </a>
-                                        <% } %>
-                                    <% } else { %>
-                                        infoCourse.json
-                                    <% } %>
-                                <% } %>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
+            <div class="card">
+            <div class="card-header bg-primary text-white d-flex mb-2">
+                <h2 class="h4">
+                    Course Settings
+                </h2>
+            </div>
```

Comment:
This is not similar to the institution settings page that we were using for reference. However, I used a card and a blue header to be more cohesive with the rest of the instructor pages. I can change it to use no card and a black text header but thought this looked better. Let me know what you think.